src/Components/test/E2ETest/Tests/VirtualizationTest.cs

```diff
@@ -183,7 +183,12 @@ public void CancelsOutdatedRefreshes_Async()
             Browser.Equal("0", () => cancellationCount.Text);
 
             // Validate that scrolling causes cancellations
-            Browser.ExecuteJavaScript("document.getElementById('async-container').scrollTo({ top:5000, behavior:'smooth' })");
+            for (var y = 1000; y <= 5000; y += 1000)
+            {
+                js.ExecuteScript($"document.getElementById('async-container').scrollTo({{ top: {y} }})");
+                Browser.Equal(y, () => (long)js.ExecuteScript("return document.getElementById('async-container').scrollTop"));
```

Comment:
This line is just about waiting until we can observe the browser did what we asked.